ui/src/apps/dashboardSettings/RootComponent.js

```diff
@@ -5,7 +5,6 @@ import _ from 'lodash'
 import { ALL_LANGUAGES } from '@/utils/i18n'
 import { withTranslation } from 'react-i18next'
 
-@Form.create()
```

Comment:
If we just remove the `@Form.create()`, no side-effects will happen?

Reply:
https://ant.design/components/form/v3/ as the doc said, `Form` no longer need to create.